src/components/HumanResources/HumanResourcesSingleStudent/HumanResourcesSingleStudent.tsx

```diff
@@ -1,28 +1,66 @@
-import React, { useState } from 'react';
+import React, { CSSProperties, useState } from 'react';
 import HumanResourcesStudentsInformation from '../HumanResourcesStudentsInformation/HumanResourcesStudentsInformation';
 import { SubmitButton } from '../../../components/common/Button/SubmitButton';
+import { useAddToTalkMutation } from '../../../api/addToTalkApi';
 
 interface Props {
+  id: string;
   firstName: string;
   lastName: string;
 }
 
 const HumanResourcesSingleStudent = (props: Props) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
-  const { firstName, lastName } = props;
+  const { id, firstName, lastName } = props;
+  const [addToTalk, { isLoading, error }] = useAddToTalkMutation();
+  const [isAdded, setIsAdded] = useState(false);
 
   const handleOpen = () => {
     setIsOpen(!isOpen);
   };
 
+  const handleBookTalk = (id: string): void => {
+    (async () => {
+      console.log(`Add to talk: ${id}`);
+
+      try {
+        const response = await addToTalk({ id });
+
+        // @ts-ignore
+        if (response.error) {
```

Comment:
widzę że nie tylko ja się ratowałem @ts-ignore :) ,

Reply:
;) to już czas goni